EnglishTranslation-master/SKILL.tsv

```diff
@@ -4597,15 +4597,15 @@ SKILL_20150714_004596	Increases the damage dealt on an enemy with [Concentrated
 SKILL_20150714_004597	Increases the damage dealt on an enemy with [Caracole] by 1% per attribute level.
 SKILL_20150714_004598	Increases the damage dealt on an enemy with [Limacon] by 1% per attribute level.
 SKILL_20150714_004599	Increases the damage dealt on an enemy with [Retreat Shot] by 1% per attribute level.
-SKILL_20150714_004600	Gun Mastery: Accuracy
+SKILL_20150714_004600	Pistol Mastery: Accuracy
```

Comment:
This should be reverted back to Gun. The Musketeer who uses rifles has this attribute and gun refers to more guns in general. 

I will check the rest of your pr later.